infermedica_api/models/diagnosis.py

```diff
@@ -127,12 +127,12 @@ def __add_evidence(self, collection, _id, state, time, initial):
         if time:
             evidence['observed_at'] = time
 
-        if initial:
-            evidence['initial'] = True
+        if source:
+            evidence['source'] = source
```

Comment:
IMO I would consider validation of the source parameter. We can do it in a similar way as we did with filter/type parameter in search method: https://github.com/infermedica/python-api/blob/master/infermedica_api/webservice.py#L163-L204

Reply:
That was exactly my thought too. However I didn't want validate API beforehand and allow to raise `Bad request 400` exception in case of any tricky logic. On the other hand validation is always a good idea, since we don't need to call API with bad request.

Reply:
I don't have a preference here 🤷